mappings/net/minecraft/util/ArraySet.mapping

```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/class_4706 net/minecraft/util/ArraySet
+	CLASS class_4707 Iterator
```

Comment:
This name will look a bit ugly in regular decompiled code like `class Iterator implements java.util.Iterator`

Reply:
I would note `java.util.AbstractList` uses `Itr` for its internal Iterator implementation.

Reply:
I don't like that name, tbh. Seems to be standard in the java libs as well, so I guess I'll take it.

Reply:
so does this directly impl iterator do does it extend abstractlist itr? iterator is fine if this extends itr